fdm-app/app/lib/email.server.ts

```diff
@@ -104,5 +104,6 @@ export async function sendMagicLinkEmailToUser(
     magicLinkUrl: string,
 ): Promise<void> {
     const email = await renderMagicLinkEmail(emailAddress, magicLinkUrl)
+    console.log(`Sending magic link ${magicLinkUrl} email to ${emailAddress}`)
     await sendEmail(email)
```

Comment:
console log delete?

Reply:
Will be resolved in fc1efe10d63570411e8ee9b49dac56c9b0e65c9f